sweep/weight_estimator.go

```diff
@@ -83,6 +83,12 @@ func (w *weightEstimator) addP2TROutput() {
 	w.estimator.AddP2TROutput()
 }
 
+// addP2WSHOutput updates the weight estimate to account for an additional
+// segwit v0 P2WSH output.
+func (w *weightEstimator) addP2WSHOutput() {
+	w.estimator.AddP2WSHOutput()
```

Comment:
Can we easily extend the `weight_estimator_test.go` tests to account for this new case? 

Alternatively, can we reproduce that sweep all txn coming in at less than 1 sat/byte scenario in the itest? One way would possibly to use a neutrino node and feed it a low estimate to trigger the case.

Reply:
Yes, but if we broadcast it in a test, btcd won't pick it up on regtest due to some config params